chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
I'm talking about something like this:

```javascript
function myFunction(objs) {
    var obj;
    while (obj = objs.shift()) {
        [...]
    }
    
    // a bunch of code

    var obj;
    while (obj = otherObjs.shift()) {
        [...]
    }
}
```
With `let`, the code at the bottom would throw an error because `obj` is reused, even though they're both temporary variables closely tied to their sections of code. You have to either 1) use a different variable name below, even if you're purposely following a similar code pattern, or 2) keep track of the usage above, which means leaving out the `let` below (which seems undesirable for the same reasons that declaring variables at the top of the function is undesirable) but being sure to add it back in if the above block is removed (though "use strict;" would at least help with that).